src/EnergyPlus/WaterCoils.cc

```diff
@@ -2875,9 +2871,10 @@ void SizeWaterCoil(EnergyPlusData &state, int const CoilNum)
             Real64 sizedMaxWaterVolFlowRate = sizerHWWaterflow.size(state, TempSize, ErrorsFound);
             // Check if the water flow rate is defined in parent HVAC equipment and set water coil design water flow rate accordingly
             if (state.dataSize->CurZoneEqNum > 0) {
-                if (ZoneEqSizing(state.dataSize->CurZoneEqNum).DesignSizeFromParent) {
-                    state.dataSize->DataWaterFlowUsedForSizing = ZoneEqSizing(state.dataSize->CurZoneEqNum).MaxHWVolFlow;
-                    state.dataWaterCoils->WaterCoil(CoilNum).MaxWaterVolFlowRate = ZoneEqSizing(state.dataSize->CurZoneEqNum).MaxHWVolFlow;
+                auto const &ZoneEqSizing = state.dataSize->ZoneEqSizing(state.dataSize->CurZoneEqNum);
+                if (ZoneEqSizing.DesignSizeFromParent) {
+                    state.dataSize->DataWaterFlowUsedForSizing = ZoneEqSizing.MaxHWVolFlow;
+                    state.dataWaterCoils->WaterCoil(CoilNum).MaxWaterVolFlowRate = ZoneEqSizing.MaxHWVolFlow;
```

Comment:
Here too. What I do in this case is global search-replace `state.dataWaterCoils->WaterCoil(CoilNum)` with `waterCoil`, compile, and then let the compiler errors show me where the shortcut needs to be defined. Recently, I have started defining all shortcuts as `auto const &` and then then letting the compiler tell me which ones should not be `const`.